README.md

```diff
@@ -317,29 +317,6 @@ sudo apt install libpixman-1-0
 All the accessible versions are stored in [this website](https://pkgs.org/download/libpixman-1-0).
 You can eventually run `apt-get check` to check for broken dependencies.
 
-
-### Installing `rl_benchmarks` package
-
-Create a dedicated conda environment:
-
-```bash
-conda create -n rl_benchmarks python=3.8
-conda activate rl_benchmarks
-````
-
-Install `rl_benchmarks` package and its dependencies using the `install.sh` file.
-
-```bash
-cd /workspace
-git clone https://github.com/owkin/rl_benchmarks_release.git
```

Comment:
Actually, `rl_benchmarks` is still the name of the package once installed (HistoSSLScaling is the name of the repo). So my suggestion would only be to replace `rl_benchmarks_release` by `HistoSSLScaling` and keep the section. Nice catch, thanks !

Reply:
Sure I can make it =)

Reply:
Perfect. Thanks a lot !

Reply:
updated, and I would suggest having the install as single line so any eventual version compatibilities are resolved correctly